LuaUI/Widgets/cmd_orbit.lua

```diff
@@ -83,3 +120,20 @@ function widget:CommandNotify(cmdID, params, options)
 		end
 	end
 end
+
+function widget:Initialize()
+	WG.Orbit = { -- add compatibility functions to a table in widget globlals
+                IsUnitGuarding = function(id)
+			local cmdQueue = Spring.GetUnitCommands(id, 1);
+
+                        if (#cmdQueue == 0) then
+				return false
+			end
+
+			local cmd = cmdQueue[1].id
+			return cmd == CMD_ORBIT or cmd == CMD_ORBIT_DRAW or cmd == CMD_GUARD or cmd == CMD_AREA_GUARD
```

Comment:
If you only look at the 1st command, prefer `Spring.GetUnitCurrentCommand` since that doesn't allocate two tables. Also, conventionally unit and command ID vars are called `unitID` and `cmdID`:
```suggestion
                IsUnitGuarding = function(unitID)
			local cmdID = Spring.GetUnitCurrentCommand(unitID)
			return cmdID == CMD_ORBIT or cmdID == CMD_ORBIT_DRAW or cmdID == CMD_GUARD or cmdID == CMD_AREA_GUARD
```